content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5473,7 +5473,10 @@ $connect(): Promise<void>
 ### <inlinecode>$on()</inlinecode>
 
 <Admonition type="warning">
-  `$on` is not available in [extended clients](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions). Please either migrate to client extensions or use the `$on` method prior to extending your client.
+  `$on` is not available in [extended
+  clients](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions).
+  Please either migrate to client extensions or use the `$on` method prior to
+  extending your client.
```

Comment:
weird wrapping again

Reply:
Already fixed: https://prisma-company.slack.com/archives/C5Z9TH6N9/p1687355406337999
In https://github.com/prisma/docs/pull/4949